admin/controllers/restart.js

```diff
@@ -4,11 +4,25 @@ const ValidationError = require('../lib/validation-error')
 
 const controller = {}
 
-controller.getRestartOverview = (req, res) => {
+controller.getRestartOverview = async (req, res, next) => {
   res.locals.pageTitle = 'Restarts'
   req.breadcrumbs(res.locals.pageTitle)
+  let restarts
+  try {
+    restarts = await restartService.getSubmittedRestarts(req.user.School)
+  } catch (error) {
+    return next(error)
+  }
+  let { hl } = req.query
+  if (hl) {
+    hl = JSON.parse(hl)
+    hl = typeof hl === 'string' ? JSON.parse(hl) : hl
```

Comment:
why JSON instead of a comma delimited query string?
(i know this isn't the first occurrence of this :) )